test/unit/account.service.stubs.ts

```diff
@@ -26,5 +26,13 @@ export const getCredentialOutput: GetCredentialIdByEmailOutput = {
   parentProfile: {
     id: faker.string.uuid(),
     fullname: faker.person.fullName(),
+    childrens: [
+      {
+        id: faker.number.int(),
+        fullname: faker.person.fullName(),
+        birthdate: faker.date.birthdate(),
+        gender: ('male' || 'female') as Genders,
```

Comment:
Tenta assim, assim ele seleciona randomicamente um dos dois, ao invés de selecionar sempre o 'male'

```suggestion
        gender: ('male' || 'female') as Genders,
```
```suggestion
        gender: faker.helpers.enumValue(Genders),
```